### ISSUE TITLE
geld verdiene mit deinem mod

### ISSUE THREAD

**Issue by DerMicha23** [Mogli12/KeyboardSteer#128]

**Comment by cowboyjo** [comment]
ist das jetzt um Stefan mit Geld zu unterstützen oder will sich da jemand mit dem Mod bereichern?
Ich habe noch nie eine Modseite gesehen, auf der man für Mods zahlt...

**Comment by Mogli12** [comment]
Würde ich den Mod hier bei GitHub für alle zur Verfügung stellen, wenn ich Geld dafür haben wollte?

**Comment by cowboyjo** [comment]
Nee, ich dachte da an einen über motivierten Fan, der dir die Kohle dann zu schiebt

**Comment by dl1esk** [comment]
Finde es auch echt übel das die damit Geld verdienen wollen, hätte ich gewusst das Mario nicht der Entwickler ist hätte ich ihn nicht gekauft. Das nenn ich mal ne Sauerrei

**Comment by cabmac** [comment]
machen die doch schon jahrelang die Spezies... wer da immernoch drauf reinfaellt...

**Comment by dl1esk** [comment]
Ich habe bei Paypal direkt einen Fall eröffnet und dies beanstandet.
Die Datei ist nun kostenlos bei denen gelistet und es wurde auch ein Credit eingefügt...

**Comment by Mogli12** [comment]
Die Datei dort zum Download anzubieten ist trotzdem Schwachsinn, da ich sie mitunter mehrmals täglich ändere.

**Comment by dl1esk** [comment]
Ja natürlich, ich denke deren Idee war damit Geld abzugreifen...

**Comment by dl1esk** [comment]
@Mogli12 ist es eigentlich möglich die Einstellungen zu personalisieren?!
Also das jeder seine eigenen Einstellungen für jedes Fahrzeug hat.
Im Moment ist es ja so das die Einstellung Global pro Fahrzeug für alle auf dem Server gilt.
